packages/app/src/cli/commands/app/logs.ts

```diff
@@ -21,8 +21,22 @@ export default class Logs extends Command {
 
   static description = this.descriptionWithoutMarkdown()
 
-  static flags = {
-    ...Dev.flags,
+  static flags: any = {
```

Comment:
Had to typecast this to `any` because the filter function I have added to filter out the undesired flags out of `dev.flags` makes it not see the `client-id` or `api-key` keys and throw a type error later in the code where we try to obtain these keys from `flags`.